module-2/lab-matplotlib-seaborn/your-code/challenge-3.ipynb

```diff
@@ -948,30 +948,12 @@
   },
   {
```

Comment:
- Well done setting categorical variables, Embarked column can also be considered a categorical variable. 
- "Cambia la frequencia de distribucion, mientras menos Bins mayor rango de edades agrupa. Es bastante subjetivo cuantos bins elegir, dependerá de lo que uno quiera identificar o transmitir". This is not too accurate, here are some general rules to set the bins for statistics purposes: 
There are a few general rules for setting bins:
1. Bins should be all the same size. For example, groups of ten or a hundred.
Bins should include all of the data, even outliers. If your outliers fall way outside of your other data, consider lumping them in with your first or last bin. This creates a rough histogram —make sure you note where outliers are being included.
2. Boundaries for bins should land at whole numbers whenever possible (this makes the chart easier to read).
Choose between 5 and 20 bins. The larger the data set, the more likely you’ll want a large number of bins. For example, a set of 12 data pieces might warrant 5 bins but a set of 1000 numbers will probably be more useful with 20 bins. The exact number of bins is usually a judgment call.
3. If at all possible, try to make your data set evenly divisible by the number of bins. For example, if you have 10 pieces of data, work with 5 bins instead of 6 or 7.
- " Grafico un Box plots para la edad, esto nos permite identificar facilmente la mediana, el IQR y los outliers". Did you mean a distribution plot?
- There is another useful plot in seaborn for plotting "counts" instead of using value_counts, it's the countplot:
```Python
sns.countplot(x='your_variable',data=data)
```
- All this about the age plot is correct
"The median is about 28,50% of the people age is between 22 and 37 aprox. There is a good amount of people that were 60+ too." There is another useful information that boxplot throws, the symmetry of the distribution. In this case,  age boxplot shows that is nearly a symmetrical distribution. 
- Plot the Fare and Age the other way around and take a look on the results of the scatter plot. 
- It's great to plot a correlation matrix as a heatmap. It's a good practice to play with color palette in order to make plots more visual. It's very common to use just one color but with different hues. You can explore the cmap parameter in seaborn and how the palette color works in matplotlib. 
- "We suspect that there is a linear relationship between Fare and Age. Use the appropriate plot to show the relationship between these 2 features. There are 2 ways, please do it both ways." You can also use a seaborn jointplot, it is usually very useful. 
```Python
sns.jointplot(x='Fare',y='Age',data=titanic)
```